stories/index.tsx

```diff
@@ -7,12 +7,13 @@ import getContext from 'recompose/getContext';
 import withContext from 'recompose/withContext';
 import withHandlers from 'recompose/withHandlers';
 import withState from 'recompose/withState';
-import { Provider, connect } from 'react-redux';
+import { Provider } from 'react-redux';
 import { createStore } from 'redux';
 import { createSelector } from 'reselect';
 import _ from 'lodash';
 
 import GenericGriddle, { actions, components, selectors, plugins, utils, ColumnDefinition, RowDefinition } from '../src/module';
+const { connect } = utils;
```

Comment:
I don't think it's necessary but may be good to use Griddle's connect here like this change introduces. That way if there is something that is going astray there it may be a little more clear. :+1: